sparkpost/transmissions.py

```diff
@@ -39,7 +39,13 @@ def _translate_keys(self, **kwargs):
             kwargs.get('use_draft_template', False)
         model['content']['reply_to'] = kwargs.get('reply_to')
         model['content']['subject'] = kwargs.get('subject')
-        model['content']['from'] = kwargs.get('from_email')
+        if kwargs.get('from_name'):
```

Comment:
Thank you very much for the PR. 

You seem have some trailing whitespaces which causing the build failure due to linting issue. Just removing the trailing whitespaces should make the build green, i hope :).